### PR TITLE
revert BUILD changes at the end of DRY_RUN=n copy.sh

### PR DESCRIPTION
**Pull request by mikedanese**
Fixes https://github.com/kubernetes/kubernetes/issues/44567

cc @caesarxuchao @ixdy @spxtr

### CODE REVIEW

File: staging/copy.sh [hunk 1]
```diff
@@ -232,4 +232,5 @@ echo "move to the client repo"
 if [ "${DRY_RUN}" = false ]; then
   ls "${CLIENT_REPO}" | { grep -v '_tmp' || true; } | xargs rm -rf
   mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
+  "${KUBE_ROOT}/hack/update-bazel.sh"
```
Comment (caesarxuchao) [inline staging/copy.sh]: Could you scope it to just affect client-go?

**Comment by mikedanese** [comment]
@caesarxuchao I'm just going to revert the changes to BUILD files in staging after the copy is compelete. PTAL

File: staging/copy.sh [hunk 2]
```diff
@@ -232,4 +232,5 @@ echo "move to the client repo"
 if [ "${DRY_RUN}" = false ]; then
   ls "${CLIENT_REPO}" | { grep -v '_tmp' || true; } | xargs rm -rf
   mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
+  git checkout HEAD -- $(find "${KUBE_ROOT}/staging/src/" -name BUILD)
```
Comment (caesarxuchao) [inline staging/copy.sh]: Can you scope it to staging/src/k8s.io/client-go? This file is in the wrong directory, it's for client-go only.

**Comment by caesarxuchao** [comment]
One request, otherwise lgtm. Thanks, @mikedanese

**Comment by mikedanese** [comment]
@caesarxuchao done.

**Comment by caesarxuchao** [comment]
Thanks, @mikedanese.

**Comment by cblecker** [comment]
```E0417 12:46:20.013] Build timed out
E0417 12:46:20.014] Build failed
```
@k8s-bot verify test this

**Pull request merged**